rust/decentraland-godot-lib/src/scene_runner/components/animator.rs

```diff
@@ -32,8 +34,8 @@ pub fn update_animator(scene: &mut Scene, crdt_state: &mut SceneCrdtState) {
                 continue;
             }
 
-            let node = godot_dcl_scene.ensure_node_mut(entity);
-            let animation_player = get_animation_player(node);
+            let (godot_entity_node, _node_3d) = godot_dcl_scene.ensure_node_3d(entity);
```

Comment:
If Im not wrong. It can be just _ instead of _node_3d to ignore the variable

Reply:
you're right, this results after running `cargo fix` and/or `cargo clippy --fix` :)